doxygen/aliases

```diff
@@ -342,6 +348,7 @@ ALIASES += ref_rfc20110614="<a href=\"https://\RFCURL/RFC_h5dump_refactor_v3.doc
 ALIASES += ref_rfc20110329="<a href=\"https://\RFCURL/RFC_Tools_Extlink_Cache_v3_r2.docx.pdf\">Support External Link Open File Cache in HDF5 Tools</a>"
 ALIASES += ref_rfc20110118="<a href=\"https://\RFCURL/RFC%20for%20h5diff%20Attribute%20Comparisons_v7.docx.pdf\"><tt>h5diff</tt> Attribute Comparisons</a>"
 ALIASES += ref_rfc20101122="<a href=\"https://\RFCURL/RFC_swmr_timeouts_v2.docx.pdf\">SWMR Timeouts</a>"
+ALIASES += ref_rfc20101119="<a href=\"https://\PUBDOCURL/2010-11_FUSEPerformanceReport-final.pdf\">FUSE Performance Report</a>"
```

Comment:
Anything in RFC section should really be in RFCURL for consistent access and when a better method is introduced it will simplify the changes.